doc_user/UserManual.adoc

```diff
@@ -828,6 +829,409 @@ image:image251.png[success]
++
+NOTE: This button is enabled only for products/lots where the column *New Lot* has the value *N*.
+
+* *[.underline]##D##elete*: this button is used to remove one or more products/lots from the list.
+* *Clean [.underline]##T##able*: this button is used to remove all the inventory rows.
+* *[.underline]##V##alidate*: this button is used to validate the inventory.
+* *[.underline]##C##onfirm*: this button is used to confirm the inventory.
++
+NOTE: The *Confirm* button is enabled only after the inventory is validated. See <<inventory-main-characteristics,Main Characteristics>> chapter.
+
+* *[.underline]##C##lose*: this button is used to return to the *_Inventory Browser_* window.
+
+[#insert-inventory-row]
+====== 4.4.2.1.1 Insert Inventory Row By Using The Code Or Description
+ 
+To insert an inventory row for a medical, type into the *Add medical* field the code or the description of the pharmaceutical and press ENTER. This will open the _Choose a Medical_ window.
+
+image:image254.png[Add a medical]
+
+Select the desired pharmaceutical or search the pharmaceutical by code or description using the search field and select the pharmaceutical to highlight it, then click on *[.underline]##S##elect* button to add the pharmaceutical as inventory row or click on *[.underline]##C##ancel* to return to the *_Inventory Browser_* window.
+
+====== 4.4.2.1.2 Insert Inventory Rows By Type And Usage
+
+To insert an inventory row for a set of medicals, useful for rotational inventories, click the *Add medicals by* button. This will open the *_Lot Information_* window.
+
+image:image255.png[Lot Information]
+
+Specify the Medical Type in the combo box (All = all medical types) and choose:
+
+* *All*: if all the pharmaceuticals for the selected Medical Type are needed.
+* *Only non-zero quantity*: if only the pharmaceuticals for the selected Medical Type with non-zero theoretical quantity are needed.
+* *With movement only*: if only the pharmaceuticals for the selected Medical Type which have been used at least once are needed.
+
+Then click on *[.underline]##O##k* button to insert all the pharmaceuticals that match the choice as the inventory(ies) row(s) in the *Inventory row table*.
+
+====== 4.4.2.1.3 Insert Inventory Row With New Lot
+
+If the pharmaceutical is already in the Inventory with its all possible lots, the system asks if to add it anyway:
+
+image:image255b.png[Add a medical]
+
+By clicking on the *[.underline]##Y##es* button, the pharmaceutical will be then added to the list with a *N* flag:
+
+image:image255c.png[New lot flag]
+
+This means that the lot has to be defined within the inventory and this can be achieved by selecting the inventory row and by clicking on the *[.underline]##L##ot* button:
```

Comment:
```suggestion
This means that the lot must be defined within the inventory.  This can be achieved by selecting the inventory row and then clicking  the *[.underline]##L##ot* button:
```